client/app/src/main/java/com/example/criengine/Profile.java

```diff
@@ -1,36 +1,47 @@
 package com.example.criengine;
 
-import java.util.Map;
+import java.util.ArrayList;
 
 public class Profile {
 
+    private String userID;
+    private String email;
     private String username;
     private String phone;
     private String firstname;
     private String lastname;
-    private String email;
     private String bio;
+    private ArrayList<String> booksOwned;
+    private ArrayList<String> booksBorrowedOrRequested;
+
+
+    public Profile() {
+        this.booksOwned = new ArrayList<>();
+        this.booksBorrowedOrRequested = new ArrayList<>();
+    }
 
-    public Profile(String username, String phone, String firstname, String lastname, String email) {
+    public Profile(String userID, String email, String username, String phone, String firstname, String lastname, String bio) {
+        this.userID = userID;
+        this.email = email;
         this.username = username;
         this.phone = phone;
         this.firstname = firstname;
         this.lastname = lastname;
-        this.email = email;
-    }
-
-    public Profile() {
-
+        this.bio = bio;
+        this.booksOwned = new ArrayList<>();
+        this.booksBorrowedOrRequested = new ArrayList<>();
```

Comment:
these two should include the type String to match the declaration (probably should be a list of book objects though)

Reply:
also we should ideally be doing some input validation to ensure that things are formatted correctly